src/views/layouts/main.hbs

```diff
@@ -14,8 +14,8 @@
     <!--<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.20/css/jquery.dataTables.min.css"/>
     <script type="text/javascript" src="https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js"></script>-->
 
-    <!-- BOOTSTRAP - BOOTSWATCH THEME - SPACELAB -->
-    <link rel="stylesheet" href="https://bootswatch.com/4/spacelab/bootstrap.min.css">
+    <!-- BOOTSTRAP - BOOTSWATCH THEME - LITERA -->
+    <link rel="stylesheet" href="https://bootswatch.com/4/litera/bootstrap.min.css">
```

Comment:
why did u change this?

Reply:
We changed this becuse Litera is the current style in SCE